util/exercise/src/cmd/configure.rs

```diff
@@ -302,7 +302,8 @@ pub fn configure_exercise(exercise_name: &str) {
     fs::write(
         &config_path,
         serde_json::to_string_pretty(&config_content).unwrap(),
-    ).expect("Failed to write the updated track configuration to the config.json file");
+    )
+    .expect("Failed to write the updated track configuration to the config.json file");
```

Comment:
This (and the similar related changes throughout this PR) doesn't look like what `rustfmt` generates, and in any case, it's unrelated to the change which is the main thrust of the PR. Please undo these. Simply running `rustfmt` should accomplish that.